command/monitor.go

```diff
@@ -0,0 +1,88 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/jrasell/levant/levant"
+	"github.com/jrasell/levant/levant/structs"
+	"github.com/jrasell/levant/logging"
+)
+
+// MonitorCommand is the command implementation that allows users to monitor
+// a deploy until it completes
+type MonitorCommand struct {
+	args []string
+	Meta
+}
+
+// Help provides the help information for the template command.
+func (c *MonitorCommand) Help() string {
+	helpText := `
+Usage: levant monitor [options] [EVAL_ID]
+
+  Monitor a Nomad job until it completes
+
+Arguments:
+
+  EVAL_ID nomad job evaluation
```

Comment:
More of a design question and I am happy to be wrong, but would it not be better if this was the JobID and then Levant would discover the allocs from this, rather than the eval ID?